docs/userGuide/syntax/code.md

```diff
@@ -281,7 +281,17 @@ A _wrap text_ button can be added to code blocks using the `codeBlockWrapButtons
 ##### Printing optimization
 
 <div id="code-print-optimization">
-Markbind enhances the readability of your code blocks for printing by applying soft wrapping, ensuring code doesn't get cut off. Additionally, line numbers are added to maintain context when wrapping occurs.
+Markbind enhances the readability of your code blocks for printing by 
+
+- Applying soft wrapping, ensuring code doesn't get cut off
+- Adding line numbers to maintain context when wrapping occurs
+- Automatically changing dark code theme to light
+- Removing [codeBlockCopyButtons]({{ baseUrl }}/userGuide/formattingContents.html#copy-button) and [codeBlockWrapButtons]({{baseUrl}}/userGuide/formattingContents.html#wrap-text-button)
+
+<box type="warning" seamless> 
+    There are some issues with printing in Firefox. Please use other browsers if these issues persist.
```

Comment:
I feel it could be better to specify suggested browser(s) e.g. Chrome as it may give the user more certainty when looking for an alternative browser if they are using Firefox